Getting-Started/Setup/Server-Setup/Load-Balancing/flexible-advanced.md

```diff
@@ -86,3 +88,31 @@ the [Explicit master scheduling server](#explicit-master-scheduling-server) conf
 Now that your front-end servers are using your custom `FrontEndReadOnlyServerRegistrar` class, they will always be deemed 'Slave' servers and will not 
 attempt any master election or task scheduling and because you are no longer using the default `DatabaseServerRegistrar` they will not try to ping
 the umbracoServer table.
+
+## Controlling how often the load balancing instructions from the database are processed and pruned
+
+During start up the `DatabaseServerMessengerOptions` can be adjusted to control how often the load balancing instructions from the database are processed and pruned.
+
+e.g. This example should be added within a [`ApplicationStarting`](../../../../Reference/Events/Application-Startup.md#startup-methods) event
```

Comment:
Is this not normally put in ApplicationStarted instead of ApplicationStarting? 🤔

Reply:
Not this one, it won't work if added to ApplicationStarted